tests/test_data.py

```diff
@@ -39,8 +39,7 @@ def test_conversor(self):
                       ConversorStub(str if arff.PY3 else unicode)]
 
         fixture = u'Iris,3.4,2,Setosa'
-        self.data.decode_data(fixture, conversors)
-        result = self.data.data[0]
+        result, = self.data.decode_rows([fixture], conversors)
```

Comment:
Just to double-check: the `result,` statement on the left makes sure the generator is exhausted by trying to create a tuple?

Reply:
I don't think it creates a tuple as such, but yes, the unpacking syntax is effectively an assertion about the number of elements on the right hand side